_inc/client/state/at-a-glance/index.js

```diff
@@ -1,2 +1,6 @@
-export * from './reducer';
```

Comment:
For reference, this change is only necessary because of this long-standing Babel bug: http://phabricator.babeljs.io/T2877